components/meter/meter-linear.js

```diff
@@ -32,23 +49,61 @@ class MeterLinear extends LitElement  {
 				position: absolute;
 				top: 0;
 			}
+
+			.d2l-meter-linear-text {
+				color: var(--d2l-color-ferrite);
+				display: flex;
+				flex-direction: row;
+				justify-content:space-between;
+				margin: 0 6px;
+			}
+
+			.d2l-meter-linear-secondary {
+				align-self: flex-end;
+			}
+
 		`];
 	}
 
 	constructor() {
 		super();
-		this.value = 0;
 		this.max = 100;
+		this.percent = false;
+		this.singleLine = false;
+		this.value = 0;
 	}
 
 	render() {
 		const progressBarWidth = this.value / this.max * 100;
+		const primary = this.percent ? `${Math.floor(this.value / this.max * 100)}%` : `${this.value}/${this.max}`;
```

Comment:
Perhaps unexpected, if `max=0`, this would produce `NaN`.  Same for the line above.

Also, looks like `${Math.floor(this.value / this.max * 100)}%` could be replaced with `${Math.floor(progressBarWidth)}%`.  Though, "width" makes me think it may not be unit-less, but those are included further below.

Reply:
Rename of `progressBarWidth` to `percent` will make this an easy substitution. I will need to handle the case where `max=0` missed that. In those cases I tend to make it `0%`